blaze/compute/numpy.py

```diff
@@ -267,7 +267,9 @@ def compute_up(t, x, **kwargs):
 
 @dispatch(Selection, np.ndarray)
 def compute_up(sel, x, **kwargs):
-    return x[compute(sel.predicate, {sel._child: x})]
+    predicate = compute(sel.predicate, {sel._child: x})
+    cond = getattr(predicate, 'values', predicate)
```

Comment:
I am trying to find where the ndarray gets coerced into a series for the `Like` to dispatch properly